gridgencore.py

```diff
@@ -380,6 +384,7 @@ def parse_yaml(self, yaml_content: dict, grid_file: str):
         self.stylesheet = self.read_str_from_grid("stylesheet") or ''
         self.author = self.read_str_from_grid("author")
         self.format = self.read_str_from_grid("format")
+        self.OutPath = self.grid_obj.get("outpath")
```

Comment:
`OutPath` is camel case, but python standard for fields is `snake_case` ie `out_path` (fwiw I mess this one up a lot myself, the snake_case thing is pretty python specific)